server_installation/topics/operating-mode/domain.adoc

```diff
@@ -294,4 +294,4 @@ $ domain.sh --host-config=host-master.xml
 $ domain.sh --host-config=host-slave.xml
 ----
 
-To try it out, open your browser and go to http://localhost:8080/auth.
+To test it out, open your browser and go to http://localhost:8080/auth.
```

Comment:
"To try it out" is valid IMO, so I'd remove this change. If changed it should be "To test it", not "To test it out".